Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -327,6 +415,65 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
 }
 
 /**
+ * Helper method to create _CTLine object given a CFAttributedStringRef
+ *
+ * @parameter string CFAttributedStringRef containing text and styling
+ *
+ * @return Unmutable array of _CTLine objects created with the requested parameters.
+ */
+static _CTLine* _DWriteGetLine(CFAttributedStringRef string) {
```

Comment:
>_DWriteGetLine [](start = 16, length = 14)

Can we avoid code duplication between _DWriteGetLine and _DWriteGetLines? We can create a _CTTypesetter object with the given CFAttributedStringRef and we should be able to call DWriteGetLines directly from here